youtube2zim/scraper.py

```diff
@@ -243,10 +260,17 @@ def run(self):
             shutil.rmtree(self.build_dir)
         self.make_build_folder()
 
-        logger.info("testing Youtbe credentials")
+        logger.info("testing Youtube credentials")
         if not credentials_ok():
             raise ValueError("Unable to connect to Youtube API v3. check `API_KEY`.")
 
+        if self.s3_url_with_credentials:
```

Comment:
can we have a `logger.info("testing S3 Optimization Cache credentials")` above?

Reply:
No. We shouldn't because we don't test creds if `if self. s3_url_with_credentials` is not true. So probably below that.